ush/rocoto/workflow_utils.py

```diff
@@ -201,8 +204,12 @@ def create_wf_task(task, cdump='gdas', cycledef=None, envar=None, dependency=Non
     # Add PARTITION_BATCH to all non-service jobs on Orion (SLURM)
     if get_scheduler(detectMachine()) in ['slurm'] and detectMachine() in ['ORION']:
         task_dict['partition'] = '&PARTITION_BATCH;'
-    # Add PARTITION_SERVICE to all service jobs (SLURM)
-    if get_scheduler(detectMachine()) in ['slurm'] and task in ['getic','arch','earc']:
+    # Add PARTITION_SERVICE to all service jobs on Hera (SLURM)
+    if get_scheduler(detectMachine()) in ['slurm'] and task in ['getic','arch','earc'] and detectMachine() in ['HERA','JET']:
+####    if get_scheduler(detectMachine()) in ['slurm'] and task in ['getic','arch','earc']:
+        task_dict['partition'] = f'&PARTITION_{task.upper()}_{cdump.upper()};'
+    # Add PARTITION_BATCH to all fcst jobs on JET (SLURM)
+    if get_scheduler(detectMachine()) in ['slurm'] and task in ['fcst'] and detectMachine() in ['JET']:
```

Comment:
I think you dismissed my earlier comments without answering them.

Reply:
I try to make change using your recommendation, but it created XML with many difference compare to my control. I would like to keep this version if possible.

Reply:
xml's are not committed to the repository and so it is irrelevant if they are different.  
They are different for different machines.

What matters is if the execution fails.

Reply:
I made the change from your request. Testing it now.